### ISSUE TITLE
Findings for accessibility staging review for VAMC Altoona

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#10933]
**Feedback framework**
- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

This epic will contain the accessibility staging review findings issue tickets.

Site Access in Devshop - http://altoonahealthcare.web.demo.ci.cms.va.gov/altoona-health-care/

## Point of Contact

**VFS Point of Contact:** Jennifer

**Comment by jenstrickland** [comment]
@SgardnerPMP 

With limited testing possible in the VA environment, the single issue ticket is all I have at this time. Once this is in Production, and I may properly test, I will add additional issues found to this epic.

There were design issues found, which may be documented elsewhere:
* 2020-07-09-Altoona - Read more link goes nowhere
* 2020-07-09-Altoona - if there is no phone no link should be there

![2020-07-09-Altoona-if thereis no phone no link should be there](https://user-images.githubusercontent.com/57469/87310627-37759480-c4ec-11ea-8278-a447b53973aa.png)

![2020-07-09-Altoona-Read-more-link-goes-nowhere](https://user-images.githubusercontent.com/57469/87310640-3b091b80-c4ec-11ea-8b57-b82045b8b7c3.png)

**Comment by jenstrickland** [comment]
@SgardnerPMP Once Altoona is in the production environment, please let me know and I will finish the staging review.

**Comment by jenstrickland** [comment]
@1Copenut For your visibility, the VAMC Altoona staging review is paused until the work is in the non-public facing production instance.
